src/plugins/slick.cellexternalcopymanager.ts

```diff
@@ -449,8 +449,15 @@ export class SlickCellExternalCopyManager implements SlickPlugin {
         (e.which === this.keyCodes.V && (e.ctrlKey || e.metaKey) && !e.shiftKey)
         || (e.which === this.keyCodes.INSERT && e.shiftKey && !e.ctrlKey)
       )) {    // CTRL+V or Shift+INS
+        const focusEl = document.activeElement as HTMLElement;
         const ta = this._createTextBox('');
-        setTimeout(() => this._decodeTabularData(this._grid, ta), 100);
+        setTimeout(() =>
+          {
```

Comment:
can you please move the curly braces on the same lines, meaning remove line 455 and 459. This is a TypeScript code, it's not C#, less lines is better 😉